watch.go

```diff
@@ -112,6 +143,9 @@ func (fb *Firebase) Watch(notifications chan Event) error {
 				// This event will be sent when the supplied auth parameter is no longer valid
 
 				// TODO: handle
+			case "rules_debug":
```

Comment:
When you create your own token with a "Debug: true" property. Sorry, I added this to this commit accidentally. I should have put this in another commit and log the data part. I suggest I add the logging to my next commit.